pmm-app/tests/verifyQAN_test.js

```diff
@@ -121,3 +122,99 @@ Scenario('Verify Main Metric change reflects in URL @not-pr-pipeline', async (I,
+  async (I, qanPage) => {
+    const serviceName = 'ps_5.7_0.0.0.0_1';
+    qanPage.waitForNewQANPageLoaded();
+    const countBefore = await qanPage.getCountOfItems();
+    qanPage.applyFilterNewQAN(serviceName);
+    I.seeInCurrentUrl('service_name=' + serviceName);
+    I.waitForInvisible(qanPage.fields.newQANSpinnerLocator, 30);
+    const countAfter = await qanPage.getCountOfItems();
+    qanPage.verifyChangedCount(countBefore, countAfter);
+  }
+);
+
+xScenario(
+  'PMM-T172 - Verify that selecting a filter updates the table data and URL  @not-pr-pipeline',
+  async (I, qanPage) => {
+    const environmentName = 'ps-dev';
+    qanPage.waitForNewQANPageLoaded();
+    const countBefore = await qanPage.getCountOfItems();
+    qanPage.applyFilterNewQAN(environmentName);
+    I.seeInCurrentUrl('environment=' + environmentName);
+    I.waitForInvisible(qanPage.fields.newQANSpinnerLocator, 30);
+    const countAfter = await qanPage.getCountOfItems();
+    qanPage.verifyChangedCount(countBefore, countAfter);
+  }
+);
+
+xScenario('PMM-T126 - Verify user is able to Reset All filters @not-pr-pipeline', async (I, qanPage) => {
+  const service_name = 'ps_5.7_0.0.0.0_1';
+  const environmentName = 'ps-dev';
+  qanPage.waitForNewQANPageLoaded();
+  const countBefore = await qanPage.getCountOfItems();
+  qanPage.applyFilterNewQAN(service_name);
+  qanPage.applyFilterNewQAN(environmentName);
+  const countAfter = await qanPage.getCountOfItems();
+  await qanPage.verifyChangedCount(countBefore, countAfter);
+  I.click(qanPage.fields.resetAll);
+  I.waitForVisible(qanPage.fields.resetAll + ':disabled', 20);
+});
+
+xScenario(
+  'PMM-T124 - Verify User is able to show all and show top 5 values for filter section @not-pr-pipeline',
+  async (I, qanPage) => {
+    const filterSection = 'Database';
+    qanPage.waitForNewQANPageLoaded();
+    await qanPage.verifyFiltersSection(filterSection, 5);
+    const countToShow = await qanPage.getCountOfFilters(filterSection);
+    qanPage.applyShowAllLink(filterSection);
+    await qanPage.verifyFiltersSection(filterSection, countToShow);
+    await qanPage.applyShowTop5Link(filterSection);
+    await qanPage.verifyFiltersSection(filterSection, 5);
+  }
+);
+
+xScenario(
+  'PMM-T125 - Verify user is able to Show only selected filter values and Show All filter values',
+  async (I, qanPage) => {
+    const service_name = 'ps_5.7_0.0.0.0_1';
+    const environmentName = 'ps-dev';
+    qanPage.waitForNewQANPageLoaded();
+    qanPage.applyFilterNewQAN(service_name);
+    qanPage.applyFilterNewQAN(environmentName);
+    I.waitForVisible(qanPage.fields.showSelected, 20);
+    I.click(qanPage.fields.showSelected);
+    await qanPage.verifyCountOfFilterLinks(2, false);
+    I.click(qanPage.fields.showSelected);
+    await qanPage.verifyCountOfFilterLinks(2, true);
+  }
+);
+
+xScenario('PMM-T123 - Verify User is able to search for filter value', async (I, qanPage) => {
```

Comment:
Let's defer this commit to a later time when the test is ready to be run.

Reply:
we need these tests, even if we skip them, for now, we can bring them back when new QAN is merged, and also these tests could be added to QAN migration branch